03-nlp/02-CLU/clu.http

```diff
@@ -0,0 +1,23 @@
+### CLU
+POST {{ language endpoint }}/language/:analyze-conversations?api-version=2022-10-01-preview 
```

Comment:
- Comments in `.http` files should only have one `#`. The three `###` are how the REST Client extension separates REST requests. One `#` is for comments.
- Please use camelCase for the variables, so the language endpoint should be `languageEndpoint` which is consistent to the others we have in the repo.